clock/clock.go

```diff
@@ -1,31 +1,25 @@
 package clock
 
 import (
-	"time"
+	"github.com/jonboulle/clockwork"
 )
 
-// Clock defines an interface for fetching time that may be used instead of the
-// time module.
+// Clock provides an interface that packages can use instead of directly
+// using the time module, so that chronology-related behavior can be tested.
 type Clock interface {
-	Now() time.Time
-	Sleep(d time.Duration)
+	clockwork.Clock
 }
 
-// SystemClock delegates calls to the time package.
-type SystemClock struct{}
-
-// NewSystemClock returns a SystemClock that delegates calls to the time package.
-func NewSystemClock() *SystemClock {
-	return &SystemClock{}
+// SystemClock is a wrapper around the jonboulle/clockwork.Clock interface
+// Clock provides an interface that packages can use instead of directly
+// using the time module, so that chronology-related behavior can be tested.
+type SystemClock struct {
+	clockwork.Clock
 }
 
-// Now returns the current local time.
-func (sc *SystemClock) Now() time.Time {
-	return time.Now()
-}
-
-// Sleep pauses the current goroutine for at least the duration d.
-// A negative or zero duration causes Sleep to return immediately.
-func (sc *SystemClock) Sleep(d time.Duration) {
-	time.Sleep(d)
+// NewSystemClock returns a SystemClock that delegates calls to the jonboulee/clockwork package.
+func NewSystemClock() *SystemClock {
+	return &SystemClock{
+		Clock: clockwork.NewRealClock(),
+	}
```

Comment:
Since `clockwork.NewRealClock` satisfies the `Clock` interface directly, there's no value in wrapping it. Change the return type to be the interface and return `NewRealClock()`.